addons/l10n_ro/data/account_tax_data.xml

```diff
@@ -2054,8 +2060,8 @@
             (0,0, {
                 'factor_percent': -50,
                 'repartition_type': 'tax',
-                'account_id': ref('pcg_4427'),
-                'plus_report_line_ids': [ref('account_tax_report_ro_tva_rd92')],
+                'account_id': ref('pcg_4426'),
+                'plus_report_line_ids': [ref('account_tax_report_ro_tva_rd242')],
```

Comment:
All the changes in this file imply the tax tags are going to change... But only on the account.tax.templates, not the tax instances. Don't we need to do some kind of conversion on them as well? It would mean existing databases will never benefit from these changes. So ... was it entirely wrong before, or is just an improvement ?

Reply:
@oco-odoo they were wrong before,
shall we do here migration? or is it a better way to update tax instances with the data from tax.templates?

Reply:
Hmmm... On second thought, I don't think we should do something in a migration script for this. In stable, it would require the users to manually update the module anyway, and would modify their configuration, with the risk of touching things we shouldn't. Especially if we want to do something complete and impact their accounting history. Let's keep it this way.